packages/health-check/src/db.js

```diff
@@ -5,9 +5,9 @@ const Memory = require("lowdb/adapters/Memory");
 
 if (!fs.existsSync("state")) fs.mkdirSync("state");
 
-const adapter = process.env.NODE_ENV === "production" ? new FileSync("state/state.json") : new Memory();
+const adapter = new FileSync("state/state.json");
 const db = low(adapter);
 
-db.defaults({ disabled: false, entries: [] }).write();
+db.defaults({ disabled: false, critical: [], verbose: [] }).write();
```

Comment:
I've been wondering whether it makes more sense to use `enabled` opposed to `disabled` 😬

Reply:
that, or a timestamp might be nice, seeing as that would flag true/false and give us insights in how long it's been disabled for

Reply:
setting disabled to anything truthy (a string with a timestamp is truthy) will coalesce to true so we can definitely do that